functional-tests/build.gradle

```diff
@@ -2,6 +2,7 @@ import org.apache.tools.ant.taskdefs.condition.Os
 
```

Comment:
Some version bumps, and moving some existing versions to variables.
Added code narc for use as a linter. (Have not yet run it)